frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -28,17 +28,34 @@ type FilterOption =
   | 'matchable'
   | 'no-check-in-email'
   | 'no-no-match-email'
+const filterOptionDisplay = [
+  ['no-filter', 'All Classes'],
+  ['unmatchable', 'Unmatchable'],
+  ['newly-matchable', 'Newly Matchable'],
+  ['matchable', 'Matchable'],
+  ['no-check-in-email', 'No Check In Email'],
+  ['no-no-match-email', 'No No Match Email'],
+]
+const sortOrderDisplay = [
+  ['newest-requests-first', 'Newest Requests First'],
+  ['classes-a-z', 'Classes A-Z'],
+  ['classes-z-a', 'Classes Z-A'],
+]
```

Comment:
I think you can use a different data structure for this, maybe a Map instead of an associative list. I think for this purpose, it doesn't make any difference since you're iterating through everything anyway, but in the future, the mapping could be useful. This is an easy fix, as I think Maps can just take in associative lists for initialization too.